app/src/components/staking/DelegationsOverview.vue

```diff
@@ -15,7 +15,7 @@
         <h1>Your Validators</h1>
         <TableValidators
           :validators="delegations.map(({ validator }) => validator)"
-          :delegations="delegations"
+          :delegations="delegations.filter(({ active }) => active === 'ACTIVE')"
```

Comment:
the API should send all current delegations that are not undelegating. this filter on the client side should not be necessary.

Reply:
yeah, it is not

Reply:
removed